packages/vscode-extension/WHATISNEW.md

```diff
@@ -1,4 +1,29 @@
-## 2.7.0 - Sep 17, 2021
+
+- [2.8.0 - Oct 18 2021](#280---oct-18-2021)
+- [2.7.0 - Sep 17 2021](#270---sep-17-2021)
+
+## 2.8.0 - Oct 18 2021
+
+Incremental version for Teams Toolkit with multiple bugs fixes and the following updates:
+
+Preview Features:
+- Support management of multiple environments.
```

Comment:
Can we add a simple screenshot for each item so that people will be more active to try them